Update about-admin-roles.md (#5234)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/add-users/about-admin-roles.md

|License admin    |   Assign the License admin role to users who need to assign and remove licenses from users and edit their usage location. <br/><br/> License admins also can: <br> - Reprocess license assignments for group-based licensing <br> - Assign product licenses to groups for group-based licensing  |
|Office Apps admin    |   Assign the Office Apps admin role to users who need to do the following: <br> - Use the Office cloud policy service to create and manage cloud-based policies for Office <br> - Create and manage service requests <br> - Manage the What's New content that users see in their Office apps   <br> - Monitor service health  |
|Password admin  |   Assign the Password admin role to a user who needs to reset passwords for non-administrators and Password Administrators.   |

> |Message Center reader |   Assign the Reports reader role to users who need to do the following: <br> - Monitor message center notifications <br> - Get weekly email digests of Message Center posts and updates <br> - Share message center posts <br> - Have read-only access to Azure AD services, such as users and groups|

Reviewer: ```suggestion
|Message center reader |   Assign the Reports reader role to users who need to do the following: <br> - Monitor message center notifications <br> - Get weekly email digests of message center posts and updates <br> - Share message center posts <br> - Have read-only access to Azure AD services, such as users and groups|
```

> |Power Platform admin |   Assign the Reports reader role to users who need to do the following: <br> - Manage all admin features for PowerApps, Microsoft Flow, and data loss prevention <br> - Create and manage service requests <br> - Monitor service health  |

Reviewer: ```suggestion
|Power platform admin |   Assign the Reports reader role to users who need to do the following: <br> - Manage all admin features for PowerApps, Microsoft Flow, and data loss prevention <br> - Create and manage service requests <br> - Monitor service health  |
```

> |Reports reader |   Assign the Reports reader role to users who need to do the following: <br> - View usage data and the activity reports in the Microsoft 365 admin center <br> - Access to the Power BI adoption content pack <br> - Access to sign-in reports and activity in Azure A <br> - View data returned by Microsoft Graph reporting API|

Reviewer: ```suggestion
|Reports reader |   Assign the Reports reader role to users who need to do the following: <br> - View usage data and the activity reports in the Microsoft 365 admin center <br> - Ger access to the Power BI adoption content pack <br> - Get access to sign-in reports and activity in Azure AD <br> - View data returned by Microsoft Graph reporting API|
```

> |Service support admin   |   Assign the Service Support admin role as an additional role to admins or users need to do the following in addition to their usual admin role: <br> - Open and manage service requests <br> - View and share message center posts <br> - Monitor service health   |

Reviewer: ```suggestion
|Service Support admin   |   Assign the Service Support admin role as an additional role to admins or users to need to do the following in addition to their usual admin role: <br> - Open and manage service requests <br> - View and share message center posts <br> - Monitor service health   |
```

|Office Apps admin    |   Assign the Office Apps admin role to users who need to do the following: <br> - Use the Office cloud policy service to create and manage cloud-based policies for Office <br> - Create and manage service requests <br> - Manage the What's New content that users see in their Office apps   <br> - Monitor service health  |
|Password admin  |   Assign the Password admin role to a user who needs to reset passwords for non-administrators and Password Administrators.   |
|Message center reader |   Assign the Reports reader role to users who need to do the following: <br> - Monitor message center notifications <br> - Get weekly email digests of message center posts and updates <br> - Share message center posts <br> - Have read-only access to Azure AD services, such as users and groups|
|Power Platform admin |   Assign the Reports reader role to users who need to do the following: <br> - Manage all admin features for PowerApps, Microsoft Flow, and data loss prevention <br> - Create and manage service requests <br> - Monitor service health  |

> |Reports reader |   Assign the Reports reader role to users who need to do the following: <br> - View usage data and the activity reports in the Microsoft 365 admin center <br> - Ger access to the Power BI adoption content pack <br> - Get access to sign-in reports and activity in Azure AD <br> - View data returned by Microsoft Graph reporting API|

Reviewer: ```suggestion
|Reports reader |   Assign the Reports reader role to users who need to do the following: <br> - View usage data and the activity reports in the Microsoft 365 admin center <br> - Get access to the Power BI adoption content pack <br> - Get access to sign-in reports and activity in Azure AD <br> - View data returned by Microsoft Graph reporting API|
```

> |Service Support admin   |   Assign the Service Support admin role as an additional role to admins or users to need to do the following in addition to their usual admin role: <br> - Open and manage service requests <br> - View and share message center posts <br> - Monitor service health   |

Reviewer: ```suggestion
|Service Support admin   |   Assign the Service Support admin role as an additional role to admins or users who need to do the following in addition to their usual admin role: <br> - Open and manage service requests <br> - View and share message center posts <br> - Monitor service health   |
```
